internal/handlers/comment.go

```diff
@@ -21,10 +22,11 @@ import (
 // @Success      200  {object}  todo.Comment
 // @Router       /comments/{id} [get]
 func GetComment(w http.ResponseWriter, r *http.Request) {
-	todo := todo.NewToDo()
+	db := db.New()
```

Comment:
Is there some way to move this logic somewhere on a higher level? Because It looks very strange that we need to do the same action for each route handler.

Reply:
Maybe we can create our custom router with the current router inside but extend it with `db` attribute?

Reply:
Yes, it will be a good next step to implement dependency for router.

Would you like to do this or want to see my variant?

Reply:
I'll try to do it